crates/ggml/src/tensor.rs

```diff
@@ -27,29 +42,60 @@ impl Tensor {
         self
     }
 
-    ///Gets the name of the tensor
-    pub fn get_name(&self) -> String {
+    /// Gets the name of the tensor
+    pub fn name(&self) -> String {
         let name = unsafe { self.ptr.as_ref().name };
         let mut name = name.iter().map(|&x| x as u8).collect::<Vec<_>>();
         name.retain(|&x| x != 0);
         String::from_utf8(name).unwrap()
```

Comment:
Okay, this one's not on you - I should have caught it, and if you don't fix it I will later - but this should probably use `CString::new` and `to_string_lossy`, because this isn't guaranteed to be UTF-8 (although 99.9999% of the time it will be), and `CString` can better maintain the invariants